packages/auth/src/providers/cognito/credentialsProvider/credentialsProvider.ts

```diff
@@ -34,6 +34,7 @@ export class CognitoAWSCredentialsAndIdentityIdProvider
 
 	private _credentialsAndIdentityId?: AWSCredentialsAndIdentityId & {
 		isAuthenticatedCreds: boolean;
+		idTokenAssociated?: string;
```

Comment:
Nit (idTokenAssociated reads like it should be a bool)

```suggestion
		associatedIdToken?: string;
```